### ISSUE TITLE
ClassTooLargeException when running HTTP package tests

### ISSUE THREAD

**Issue by chamil321** [ballerina-platform/ballerina-lang#32400]
**Description:**
$subject. Please find the exception below

[Workflow log.zip](https://github.com/ballerina-platform/ballerina-lang/files/7061808/logs_5915.zip)

```
> Task :http-ballerina-tests:ballerinaTest FAILED
[2021-08-26 17:52:20,686] SEVERE {b7a.log.crash} - Class too large: ballerina/http_tests/2/$_string_constants 
org.objectweb.asm.ClassTooLargeException: Class too large: ballerina/http_tests/2/$_string_constants
	at org.objectweb.asm.ClassWriter.toByteArray(ClassWriter.java:538)
	at org.wso2.ballerinalang.compiler.bir.codegen.split.JvmBStringConstantsGen.generateConstantInit(JvmBStringConstantsGen.java:114)
	at org.wso2.ballerinalang.compiler.bir.codegen.JvmPackageGen.generate(JvmPackageGen.java:829)
	at org.wso2.ballerinalang.compiler.bir.codegen.CodeGenerator.generate(CodeGenerator.java:92)
	at org.wso2.ballerinalang.compiler.bir.codegen.CodeGenerator.generateTestModule(CodeGenerator.java:73)
	at io.ballerina.projects.JBallerinaBackend.performCodeGen(JBallerinaBackend.java:320)
	at io.ballerina.projects.ModuleContext.generateCodeInternal(ModuleContext.java:426)
	at io.ballerina.projects.ModuleCompilationState$4.generatePlatformSpecificCode(ModuleCompilationState.java:132)
	at io.ballerina.projects.ModuleContext.generatePlatformSpecificCode(ModuleContext.java:345)
	at io.ballerina.projects.JBallerinaBackend.performCodeGen(JBallerinaBackend.java:159)
	at io.ballerina.projects.JBallerinaBackend.<init>(JBallerinaBackend.java:137)
	at io.ballerina.projects.JBallerinaBackend.lambda$from$0(JBallerinaBackend.java:112)
	at java.base/java.util.HashMap.computeIfAbsent(HashMap.java:1133)
	at io.ballerina.projects.PackageCompilation.getCompilerBackend(PackageCompilation.java:170)
	at io.ballerina.projects.JBallerinaBackend.from(JBallerinaBackend.java:111)
	at io.ballerina.cli.task.CompileTask.execute(CompileTask.java:80)
	at io.ballerina.cli.TaskExecutor.executeTasks(TaskExecutor.java:40)
	at io.ballerina.cli.cmd.TestCommand.execute(TestCommand.java:211)
	at java.base/java.util.Optional.ifPresent(Optional.java:183)
	at io.ballerina.cli.launcher.Main.main(Main.java:53)
```

Debugs up to certain level with @warunalakshitha and seeing followings

bStringVarMap size = 13433

<img width="1240" alt="Screenshot 2021-08-26 at 23 30 53" src="https://user-images.githubusercontent.com/8381003/131012977-8c9cac94-e780-437e-ab33-baf5a060ef59.png">

Observability constants

<img width="1237" alt="Screenshot 2021-08-26 at 23 30 05" src="https://user-images.githubusercontent.com/8381003/131013205-3c355fff-98ee-44ce-b61e-ba56b6b87699.png">

**Steps to reproduce:**
https://github.com/ballerina-platform/module-ballerina-http/pull/580/checks

**Affected Versions:**
2.0.0-beta.3-20210826-194200-fe49078f

This works fine in 2.0.0-beta.3-20210821-093200-e064dcef which used in current HTTP master.

**Comment by chamil321** [comment]
The string count has reduced with fix #32429

<img width="1536" alt="Screenshot 2021-08-28 at 22 45 38" src="https://user-images.githubusercontent.com/8381003/131225763-57f21a9f-6b1a-4717-9f37-201716e66c41.png">

**Comment by warunalakshitha** [comment]
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/32429.
